### ISSUE TITLE
Frame resizing

### ISSUE THREAD

**Issue by JSKoenig** [majikat768/HuntStatsLogger#12]
When selecting a match the ui resizes to its default size

**Comment by majikat768** [comment]
Fixed in the repo if running as a python script, I'll get a new exe posted today that should fix this as well.

**Comment by JSKoenig** [comment]
UI doesn't resize from selecting different matches but it does after a new one posts.

**Comment by majikat768** [comment]
Yep, I just found the problem, should be fixed in the repo now.  Thanks!

**Comment by majikat768** [comment]
It will still resize automatically if the new data doesn't fit in the current window.
I may change this in the future, but currently it's set to have a minimum window size, based on the window's content.
